### DISCUSSION TITLE
plugin repo settings

### DISCUSSION THREAD

**Discussion by wimglenn** [pytest-dev/pytest#12730] [General]
Hi, I transferred ownership of a plugin `pytest-freezer` from wimglenn to pytest-dev some time ago

But I can no longer see/manage settings (the https://github.com/pytest-dev/pytest-freezer/settings URL is a 404 for me now, and there is no "settings ⚙️" tab visible on https://github.com/pytest-dev/pytest-freezer 

Is that expected? How can I see/manage the project settings now?

**Comment by RonnyPfannschmidt** [comment]
I believe the setup of members was accidentally incomplete, @nicoddemus can you verify, I won't be at a computer soon
